source/_docs/scripts/conditions.markdown

```diff
@@ -9,6 +9,25 @@ Unlike a trigger, which is always `or`, conditions are `and` by default - all co
 
 All conditions support an optional `alias`.
 
+The different types of condition are:
```

Comment:
We can add `toc: true` to the frontmatter and use `{{ content | toc_only }}` instead of this to make it dynamic

Reply:
Is this for me? I only see `{{ content | toc_only }}` used in included html

AFAIK, autogenerated TOCs depends on the MD handler. Don't know which one is used here (nor which plugins would be available)

Reply:
it is available in the current markdown file. Could you try?

Reply:
I'm completely at loss regarding what you want me to do, to be honest :)

Would you have an example or some doc pointers?
Just a link to a page implementing the "toc" in question should do.

Thanks

Reply:
Its in the first comment, https://github.com/home-assistant/home-assistant.io/pull/19156/files#r700000296

Add the frontmatter option and the small liquid snippet in the place you want to add the TOC.

Reply:
Oh ok.

That's what I understood
```
diff --git a/source/_docs/scripts/conditions.markdown b/source/_docs/scripts/conditions.markdown
index 7e3590362..8161d2959 100644
--- a/source/_docs/scripts/conditions.markdown
+++ b/source/_docs/scripts/conditions.markdown
@@ -1,6 +1,7 @@
 ---
 title: "Conditions"
 description: "Documentation about all available conditions."
+toc: true
 ---
 
 Conditions can be used within a script or automation to prevent further execution. When a condition does not return true, the script or automation stops executing. A condition will look at the system at that moment. For example, a condition can test if a switch is currently turned on or off.
@@ -9,6 +10,8 @@ Unlike a trigger, which is always `or`, conditions are `and` by default - all co
 
 All conditions support an optional `alias`.
 
+{{ content | toc_only }}
+
 ## AND condition
 
 Test multiple conditions in one condition statement. Passes if all embedded conditions are valid.
```

but it doesn't do anything, and I only saw those in html, not markdown, so I assumed I misunderstood.

Reply:
I see an empty

```html
<ul id="toc" class="section-nav">
</ul>
```

in the generated html, though. So there is likely a piece missing somewhere

Reply:
@ludeeus @frenck If you could maybe apply what you suggest to https://github.com/home-assistant/home-assistant.io/blob/current/source/_docs/scripts.markdown, which also has a hardcoded TOC, it will serve as a template. Thx

Reply:
```yaml
toc: true
no_toc: true
```
...
`{{ page.content | markdownify | toc_only  }}`

Reply:
Working perfectly. Thanks.
I switched the "script" page to auto-generated TOC as well.